### DISCUSSION TITLE
Array to string conversion error

### DISCUSSION THREAD

**Discussion by stephenmeehanuk** [statamic/cms#5579] [Troubleshooting] [answered]
Hi, recently updated to `3.3.2` from `3.2.38`, using `regex` antlers at the moment.

My blog home page gives this error `Array to string conversion` after a bit of digging I found the issue was related to this:
```
Read time {{ bard | flatten | join | strip_tags | read_time  }} mins
```
The page loads when I remove that

I found the snippet above on the forum (https://statamic.com/forum/2970-read-time-from-bard-fieldtype) it's dated 2018. 

Is there a workaround for this in 3.3?

**Comment by duncanmcclean** [comment, marked as answer]
This sounds like a bug, would you be able to open an issue instead?

**Comment by stephenmeehanuk** [comment]
Done
https://github.com/statamic/cms/issues/5592
